src/models/Settings.js

```diff
@@ -312,11 +312,6 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       else if (BrowserFeatures.corsIsNotSupported()) {
         this.callGlobalError(new UnsupportedBrowserError(Okta.loc('error.unsupported.cors')));
       }
-      else if (options.features) {
-        if (options.features.identifierFirst && !options.features.idpDiscovery) {
-          this.callGlobalError(new ConfigError(Okta.loc('error.invalid.identifierFirst')));
```

Comment:
Followup for this PR or later - we should remove this property from i18n as well